rocAL/rocAL_pybind/amd/rali/pipeline.py

```diff
@@ -230,7 +239,9 @@ def define_graph(self):
         """This function is defined by the user to construct the
         graph of operations for their pipeline.
         It returns a list of outputs created by calling RALI Operators."""
-        raise NotImplementedError
+        print("define_graph NotImplemented")
+        return 0
```

Comment:
Why not raise an error here?

Reply:
This was added to have trainings & use example files of both the new API & old API simultaneously

Once we confirm that all the trainings using new API is working fine, we can revert this change